prow/plugins/releasenote/releasenote.go

```diff
@@ -26,8 +26,10 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-const pluginName = "release-note"
-
+const (
+	trustedOrg = "kubernetes"
```

Comment:
I can look into that, I just grabbed the hardcoded option since that was the first usage example I saw elsewhere in the code base. I gave the event type a coursery glance just now and it seems simple to just grab the repo owner.